indico/modules/rb/client/js/components/AdminOverrideBar.jsx

```diff
@@ -25,7 +25,11 @@ const AdminOverrideBar = ({visible, disable}) => {
   const trigger = (
     <span>
       <Icon name="exclamation triangle" />
-      <Translate>Admin override enabled</Translate>
+      <Translate>
+        Admin override enabled - Use{' '}
+        <Param name="tipShortcutComboKeyCtrl" value={<kbd>Ctrl</kbd>} /> +{' '}
```

Comment:
and use a single param for `<><kbd>Ctrl</kbd> + <kbd>A</kbd></>` (and a simple name like `"shortcut"`